lib/block-supports/spacing.php

```diff
@@ -51,11 +51,10 @@ function gutenberg_apply_spacing_support( $block_type, $block_attributes ) {
 		return $attributes;
 	}
 
-	$style_engine                    = WP_Style_Engine_Gutenberg::get_instance();
 	$spacing_block_styles            = array();
 	$spacing_block_styles['padding'] = $has_padding_support ? _wp_array_get( $block_styles, array( 'spacing', 'padding' ), null ) : null;
 	$spacing_block_styles['margin']  = $has_margin_support ? _wp_array_get( $block_styles, array( 'spacing', 'margin' ), null ) : null;
-	$inline_styles                   = $style_engine->generate(
+	$inline_styles                   = WP_Style_Engine_Gutenberg::generate(
```

Comment:
This raises an interesting question that doesn't need to be answered now though :P 

Basically here we're using the "gutenberg" name and not the core name. This works for now because we're back porting the block supports code manually, but we were considering having automatic backports (like block-library and parser) for block supports changes. So when we do that, we'll have to find a way to use the core name here and also rename it to the gutenberg name on the build step.

Reply:
> So when we do that, we'll have to find a way to use the core name here and also rename it to the gutenberg name on the build step.

Yeah, I was thinking we could have a single function somewhere that returns `WP_Style_Engine_Gutenberg::get_instance()` so we'd only have one place to update it. 🤷